step1_example1.py

```diff
@@ -0,0 +1,33 @@
+# キーボードから数字を取得
+num1 = input("ひとつめの数字を入力してください。")
+num2 = input("ふたつめの数字を入力してください。")
+
+try :
+    # 足し算
+    wa = int(num1) + int(num2)
+    print(str(num1) + " ＋ " + str(num2) + " = " + str(wa))
+
+    # 引き算
+    if num1 >= num2 :

```

Comment:
```suggestion
    if int(num1) >= int(num2) :
```
inputの戻り値はstr型なので、int型にキャストしますね。